web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.ts

```diff
@@ -128,22 +169,41 @@ export class ProjectExplorer implements OnInit {
         this.numLibraryFoldersDisplayed += Config.ELEMENT_CHUNK_SIZE;
     }
 
-    public switchToLibrary(): void {
-        this.showLibrary = true;
+    public loadMoreRecycleBinFolders(): void {
+        this.numRecycleBinFoldersDisplayed += Config.ELEMENT_CHUNK_SIZE;
     }
 
     public switchToProject(): void {
+        this.showProject = true;
+        this.showLibrary = false;
+        this.showRecycleBin = false;
+    }
+
+    public switchToLibrary(): void {
+        this.showProject = false;
+        this.showLibrary = true;
+        this.showRecycleBin = false;
+    }
+    public switchToRecycleBin(): void {
+        this.showProject = false;
         this.showLibrary = false;
+        this.showRecycleBin = true;

```

Comment:
Since we now have three different tabs (project, library, recycleBin), which are mutual exclusive, I would suggest to go with an Enum instead of three boolean flags.